nucleus/experimental/nucleus_integration.py

```diff
@@ -0,0 +1,161 @@
+# This file contains all the Core Nucleus <-> HMI integrations
```

Comment:
A couple of thoughts:
1. Maybe we should just rename this folder to `deploy`. This is just going to go into a separate release branch in the meantime anyway.
2. Regardless of what we name this folder (where the HMI/deploy stuff lives), I'd say that the integration code should go in an actual Nucleus folder. This way, it's clear that Nucleus depends on Deploy, without an inverse dependency too. Having this integration file here means that we can't just move this folder to a separate repo later, for instance.